ci/Dockerfile

```diff
@@ -35,16 +35,14 @@ RUN apk add --no-cache --no-progress --update --upgrade \
         && \
     mv /etc/apk/repositories /etc/apk/repositories.old && \
     apk add --no-cache --update --upgrade \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/community \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.12/main \
```

Comment:
This version bump appears to be an unrelated change and not mentioned in the PR description. 

* If necessary - add thy why to the PR description.
* If not necessary, move into a follow-up PR.

Reply:
removed